updates/1407_a0002_quests_a_donation_of_cloth.sql

```diff
@@ -0,0 +1,178 @@
+-- quest id 7802 and 7807 changed
+UPDATE `quest_template` SET `RewRepFaction1` = 47 WHERE `entry` = 7802;
+UPDATE `creature_involvedrelation` SET `quest` = 7802 WHERE `id` = 14723 AND `quest` = 7807;
+UPDATE `creature_questrelation` SET `quest` = 7802 WHERE `id` = 14723 AND `quest` = 7807;
+
+UPDATE `quest_template` SET `RewRepFaction1` = 54 WHERE `entry` = 7807;
+UPDATE `creature_involvedrelation` SET `quest` = 7807 WHERE `id` = 14724 AND `quest` = 7802;
+UPDATE `creature_questrelation` SET `quest` = 7807 WHERE `id` = 14724 AND `quest` = 7802;
+
+
+-- quest id 7803 and 7808 changed
+UPDATE `quest_template` SET `RewRepFaction1` = 47 WHERE `entry` = 7803;
+UPDATE `creature_involvedrelation` SET `quest`= 7803 WHERE `id` = 14723 AND `quest` = 7808;
+UPDATE `creature_questrelation` SET `quest` = 7803 WHERE `id` = 14723 AND `quest` = 7808;
+
+UPDATE `quest_template` SET `RewRepFaction1` = 54 WHERE `entry` = 7808;
+UPDATE `creature_involvedrelation` SET `quest` = 7808 WHERE `id` = 14724 AND `quest` = 7803;
+UPDATE `creature_questrelation` SET `quest` = 7808 WHERE `id` = 14724 AND `quest` = 7807;
```

Comment:
UPDATE `creature_questrelation` SET `quest` = 7808 WHERE `id` = 14724 AND `quest` = 7803;